### DISCUSSION TITLE
Question: Is there a way to transform the contents of the editor state into html?

### DISCUSSION THREAD

**Discussion by vladcl** [facebook/lexical#3451] [Q&A] [answered]
Hi, mates!

I'm using lexical editor in my application but i want to generate a pdf with the content in editor state.

i was using jsPDF + react-wysiwyg to generate the PDF, so i want to know if is possible transform the content to html to use in jsPDF lib.

Thanks!

**Comment by thegreatercurve** [comment, marked as answer]
See docs here for Lexical->HTML conversion: https://lexical.dev/docs/concepts/serialization#lexical---html
